lib/wings/models/file_node.rb

```diff
@@ -5,75 +5,80 @@ class FileNode < ::Valkyrie::Resource
     # TODO: Branch valkyrie6 included the valkyrie resource access controls.  Including this now causes an exception.
     #       Need to explore whether this line should be uncommented.
     # include ::Valkyrie::Resource::AccessControls
-    # attribute :alternate_id, ::Valkyrie::Types::Set # AF::File metadata_node id
+    # TODO: Other Resources created by wings uses the `alternate_id` to store the id of the AF related object.  This uses
+    #       `file_identifiers` instead.  May want to explore whether this inconsistency is necessary or justified as
+    #       standard valkyrie approach.
+    # attribute :alternate_id, ::Valkyrie::Types::Set
+    # TODO: Not seeing a use case for including member_ids for FileNode.  Leaving until determined for sure it is not
+    #       needed as all the other AF related resources include this.
+    # attribute :member_ids, ::Valkyrie::Types::Set
 
     attribute :file_set_id, ::Valkyrie::Types::ID
+    attribute :file_identifiers, ::Valkyrie::Types::Set
```

Comment:
Group attributes set by FileNodeBuilder together at the top of the list of attributes.